### PR TITLE
Split out SlackTeamCommander

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/slack/SlackCommander.scala [hunk 1]
```diff
@@ -61,22 +50,14 @@ class SlackCommanderImpl @Inject() (
   libToChannelRepo: LibraryToSlackChannelRepo,
   slackClient: SlackClientWrapper,
   libToSlackPusher: LibraryToSlackChannelPusher,
-  ingestionCommander: SlackIngestionCommander,
   basicUserRepo: BasicUserRepo,
   pathCommander: PathCommander,
   permissionCommander: PermissionCommander,
   orgCommander: OrganizationCommander,
-  orgDomainCommander: OrganizationDomainOwnershipCommander,
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackCommander.scala:69]: This wasn't used anywhere; expected?

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackCommander.scala:69]: yeah I removed the line that used it but I forgot about the dependency

**Comment by leogrim** [comment]
Perhaps rename `SlackCommander` to something like `SlackLibraryCommander` ?

**Comment by ryanpbrewster** [comment]
Sorry for your merge conflicts, friends.

**Pull request merged**